rs/dre-canisters/trustworthy-node-metrics/add_np.py

```diff
@@ -0,0 +1,60 @@
+import csv
+import subprocess
+
+# Define the file paths and constants
+csv_file_path = "/Users/pietro.di.marco/Documents/dfinity/dre-1/rs/dre-canisters/trustworthy-node-metrics/src/trustworthy-node-metrics/node_info_api.csv"  # Path to the CSV file
+did_file_path = "/Users/pietro.di.marco/Documents/dfinity/dre/rs/dre-canisters/trustworthy-node-metrics/src/trustworthy-node-metrics/trustworthy-node-metrics.did"
```

Comment:
If possible this shouldn't include your personal paths. Can you make this relative?